src/main/kotlin/no/skatteetaten/aurora/gobo/security/UserService.kt

```diff
@@ -18,28 +19,29 @@ class UserService {
         val ANONYMOUS_USER = User(GUEST_USER_ID, GUEST_USER_NAME)
     }
 
-    fun getCurrentUser(): User {
-        val context = SecurityContextHolder.getContext()
-        val authentication: Authentication? = context.authentication
-
+    fun getCurrentUser(request: HttpServletRequest? = null): User {
```

Comment:
Disse to getCurrentUser-metodene kan neste være extensionmethods på HttpServletRequest og DataFetchingEnvironment siden de ikke har collaborators eller state?

Reply:
Ja, godt forslag. Jeg tenkte på det samme, skal prøve det ut og se hvordan det blir.